inlong-manager/manager-dao/src/main/resources/mappers/StreamSourceEntityMapper.xml

```diff
@@ -434,15 +434,13 @@
     <update id="updateStatus">
         update stream_source
         set previous_status = status,
-            status          = #{nextStatus, jdbcType=INTEGER},
-            modify_time     = modify_time
```

Comment:
In our design, the modify_time field is used to record the user's modification time.
The system modifies the status should not let the modify_time change.